src/services/services.ts

```diff
@@ -1627,12 +1626,13 @@ namespace ts {
                 return file.sourceMapper;
             }
             let mapFileName = scanForSourcemapURL(fileName);
-            if (mapFileName && dataURLRegExp.exec(mapFileName)) {
-                const b64EncodedMatch = base64UrlRegExp.exec(mapFileName);
-                if (b64EncodedMatch) {
-                    const base64Object = b64EncodedMatch[1];
+            let match: RegExpExecArray;
+            if (mapFileName && (match = base64UrlRegExp.exec(mapFileName))) {

```

Comment:
nit: I'd just use a nested `if` to avoid having `match` declared outside of the block since it isn't used elsewhere.